tests/common.go

```diff
@@ -10110,3 +10136,27 @@ func GetRandomSubset(elements []string, subsetSize int) ([]string, error) {
 
 	return shuffledElements[:subsetSize], nil
 }
+
+// DeleteAllNamespacesCreatedByTestCase deletes all the namespaces created for the test case
+func DeleteAllNamespacesCreatedByTestCase(CurrentTestRailTestCaseId int) error {
```

Comment:
We can remove the return type error because we are not returning it anyway

Reply:
We are returning error at the first if.

Reply:
Got it